lib/trino-plugin-toolkit/src/test/java/io/trino/plugin/base/security/TestFileBasedAccessControl.java

```diff
@@ -116,6 +121,22 @@ public void testEmptyFile()
         accessControl.checkCanShowRoleGrants(UNKNOWN);
     }
 
+    @Test
+    public void testEmptyFunctionKind()
```

Comment:
should the same be provided for the System AC?
Are there other tests which should be the same in System/Connector AC?

Reply:
For completeness I add this test into `TestFileBasedSystemAccessControl` as well.